indico/modules/events/sessions/ical.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import icalendar
+from werkzeug.urls import url_parse
+
+from indico.core.config import config
+from indico.modules.events.ical import generate_basic_component
+from indico.web.flask.util import url_for
+
+
+def generate_session_component(session, related_to_uid=None):
+    """Generate an Event iCalendar component from an Indico Session."""
+    uid = f'indico-session-{session.id}@{url_parse(config.BASE_URL).host}'
+    url = url_for('sessions.display_session', session, _external=True)
+    component = generate_basic_component(session, uid, url)
+
+    if related_to_uid:
+        component.add('related_to', related_to_uid)
+
```

Comment:
This line is suspicious.  I do not say that the code is wrong, I have not run it.

The property is officially called with hyphen `RELATED-TO`, but the code adds a property with underscore `related_to`.

Reply:
Good catch, fix on the way.